frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -888,10 +888,12 @@ export const dashboardLogic = kea<dashboardLogicType>([
                     return []
                 }
 
-                const validDates = insightTiles
+                const validDates: Dayjs[] = insightTiles
                     .filter((i) => !!i.insight?.cache_target_age || !!i.insight?.next_allowed_client_refresh)
-                    .map((i) => dayjs(i.insight?.cache_target_age ?? i.insight?.next_allowed_client_refresh))
-                    .filter((date) => date.isValid())
+                    .map((i) =>
+                        dayjs.min(dayjs(i.insight?.cache_target_age), dayjs(i.insight?.next_allowed_client_refresh))
+                    )
```

Comment:
`dayjs(undefined)` means "now", so we should not include the `i.insight?.cache_target_age` if its value is undefined/null (and same for `i.insight?.next_allowed_client_refresh`)